data/blog/new-website.md

```diff
@@ -0,0 +1,191 @@
+translating on [Crowdin], where you can sign in using your GitHub account.
+
+There are currently 21 languages enabled for the Electron project on Crowdin. 
+Adding support for more languages is easy, so if you're interested in 
+helping translate but you don't see your language listed, 
+[let us know](https://github.com/electron/electron.atom.io/issues/new) and
+we'll enable it.
+
+## Raw Translated Docs
+
+If you prefer to read documentation in raw markdown files, you 
+can now do that in any language:
+
+```sh
+git clone https://github.com/electron/electron-i18n
+ls electron-i18n/content
+```
+
+## App Pages
+
+As of today, any Electron app can easily have its own page on the Electron 
+site. For a few examples, check out 
+[Etcher](https://electronjs.org/apps/etcher),
+[1Clipboard](https://electronjs.org/apps/1clipboard), or 
+[GraphQL Playground](https://electronjs.org/apps/graphql-playground), pictured
+here on the Japanese version of the site:
+
+<figure>
+  <a href="https://electronjs.org/apps/graphql-playground">
+    <img class="screenshot"  src="https://user-images.githubusercontent.com/2289/32871096-f5043292-ca33-11e7-8d03-a6a157aa183d.png" alt="GraphQL Playground">
+  </a>
+</figure>
+
+There are some incredible Electron apps out there, but they're not always easy
+to find, and not every developer has the time or resources to build a proper 
+website to market and distribute their app.
+
+Using just a 
+[PNG icon file and a small amount of app metadata](https://github.com/electron/electron-apps/blob/master/contributing.md),
+we're able to collect a lot of information about a given app.
+Using data collected from GitHub, app pages can now display screenshots, 
+download links, versions, release notes, and READMEs for every app that 
+has a public repository. Using a color palette extracted from each app's icon,
+ we're able to produce [bold and accessible colors](https://github.com/zeke/pick-a-good-color)
+to give each app page some visual distinction.
+
+The [apps index page](https://electronjs.org/apps) now also has categories,
+and a keyword filter to find interesting apps like [GraphQL GUIs](https://electronjs.org/apps?q=graphql)
+and [p2p tools](https://electronjs.org/apps?q=graphql).
+
+If you've got an Electron app that you'd like featured on the site, open a 
+pull request on the [electron/electron-apps] repository.
+
+## One-line Installation with Homebrew
+
+The [Homebrew] package manager for macOS has a subcommand called [cask]
+that makes it easy to install desktop apps using a single command in your
+terminal, like `brew cask install atom`.
+
+We've begun collecting Homebrew cask names for popular Electron apps and are now
+displaying the installation command (for macOS visitors) on every app page 
+that has a cask:
+
+<figure>
+  <a href="https://electronjs.org/apps/dat">
+   <img  class="screenshot" src="https://user-images.githubusercontent.com/2289/32871246-c5ef6f2a-ca34-11e7-8eb4-3a5b93b91007.png">
```

Comment:
Extra whitespace

Not sure about the formatting for indented lines... Probably inserting an extra space before `<img...`? Same with next line.